src/main/resources/conf/base.conf

```diff
@@ -626,8 +626,8 @@ mantis {
 
 akka {
   loggers = ["akka.event.slf4j.Slf4jLogger"]
-  # Not using ${logging.logs-level} because it might be set to TRACE, which our version of Akka doesn't have.
-  loglevel = "INFO"
+  # the effective log level is configured for Logback; this loglevel only determines what is passed to Slf4j 
+  loglevel = "DEBUG"
```

Comment:
ok so I think this overthinking it. the consequence of passing `TRACE` to the log system is:
```
[ERROR] [05/13/2021 09:51:24.512] [run-main-0] [EventStream(akka://mantis_system)] unknown akka.loglevel TRACE
akka.event.Logging$LoggerException:
	at akka.event.LoggingBus.$anonfun$startDefaultLoggers$1(Logging.scala:113)
	at akka.event.LoggingBus.$anonfun$startDefaultLoggers$1$adapted(Logging.scala:110)
	at scala.Option.getOrElse(Option.scala:201)
	at akka.event.LoggingBus.startDefaultLoggers(Logging.scala:110)
	at akka.event.LoggingBus.startDefaultLoggers$(Logging.scala:108)
	at akka.event.EventStream.startDefaultLoggers(EventStream.scala:26)
	at akka.actor.LocalActorRefProvider.init(ActorRefProvider.scala:605)
	at akka.actor.ActorSystemImpl.liftedTree2$1(ActorSystem.scala:1021)
	at akka.actor.ActorSystemImpl._start$lzycompute(ActorSystem.scala:1017)
	at akka.actor.ActorSystemImpl._start(ActorSystem.scala:1017)
	at akka.actor.ActorSystemImpl.start(ActorSystem.scala:1040)
	at akka.actor.ActorSystem$.apply(ActorSystem.scala:272)
	at akka.actor.ActorSystem$.apply(ActorSystem.scala:316)
	at akka.actor.ActorSystem$.apply(ActorSystem.scala:260)
```
I think this is fine and it's a better solution to use ${logging.logs-level} instead of trying to guess what a good level might be.
maybe a note on the `logs-level` setting that notes this interaction.

Reply:
Well, that won't work as expected when overall "logs-level" is info but you configure some specific parts for a level lower than that.

Reply:
For me this feels like a bit of personal preference. How often do we use a log level of trace anyway? For that I think I prefer Dom's perspective